src/gp/src/GPMSA.C

```diff
@@ -363,7 +363,10 @@ GPMSAEmulator<V, M>::lnValue(const V & domainVector,
     // Add small white noise component to diagonal to make stuff +ve def
     // = "small ridge"
     // Barely alluded to, never described, in Higdon et. al.
-    const double emulator_data_precision = domainVector[dimSum-1-(numOutputs>1)];
+    const double emulator_data_precision =
+      m_opts.m_calibrateObservationalPrecision ? domainVector[dimSum-2-(numOutputs>1)]
+                                               : domainVector[dimSum-1-(numOutputs>1)];
+
```

Comment:
Can we just subtract the bool here (as is done in GPMSAOptions with the same option, or here with (numOutputs>1)) rather than using a ternary operator?